tutorials/rigid-body-dynamics.qmd

```diff
@@ -0,0 +1,246 @@
+```math
+\begin{aligned}
+\dot{p} &= X \\
+\dot{X} &= X_X\\
+\dot{X_X} &= 0
+\end{aligned}
+```
+
+To solve the ODE we still need an affine connection on the manifold $(p, X, X_X)$ lives on.
+A nice way of describing that manifold in `Manifolds.jl` is by using a vector bundle where each fiber corresponds to direct sum of tangent spaces.
+There is, however, one small caveat: `X_X` was brought from the double tangent bundle and depends on the connection we (should have) selected for the tangent bundle $T\operatorname{SE}(2)$.
+So we may use the connection from $\operatorname{SE}(2)$ but we may also use the Levi-Civita connection of the Sasaki metric on $T\operatorname{SE}(2)$.
+We shall explore this choice later in the tutorial.
+
+For simplicity again we will use a first order forward Euler discretization to solve our ODE (see [technical note](@sec-forward-euler) if you disapprove).
+
+## Technical notes
+
+These technical notes elaborate on some aspects of the tutorial that a curious reader may find interesting but are not necessary to effectively work with `Manifolds.jl`
+
+### Different exponential maps and connections
+
+The exponential map we used in this tutorial is not the only one available on Lie groups.
+Another choice is the so-called ``[Lie group exponential](https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/group.html#Manifolds.exp_lie-Tuple{SpecialEuclidean,%20Any})``{=commonmark}.
+In the case of the special Euclidean group, Lie group exponential mixes the translation and rotation parts of a tangent vector:
+
+```{julia}
+ts = 0.0:0.25:1.0
+pts = [exp_lie(SE2, t * X) for t in ts]
+plt5 = plot(; aspect_ratio=:equal)
+for (t, p) in zip(ts, pts)
+    draw_arrow_at(p; plt=plt5, arrow_kwargs=(; label="arrow at t=$t"))
+end
+plt5
+```
+
+Note also that the default choice of connection for Lie groups is not the only one.
+There are also ``[Cartan-Schouten connections](https://juliamanifolds.github.io/Manifolds.jl/stable/manifolds/group.html#Cartan-Schouten-connections)``{=commonmark}.
+They give the same exponential and logarithmic maps but a different parallel transport.
+
+### Jet bundles?
+
+Yes, [📖 jet bundles](https://en.wikipedia.org/wiki/Jet_bundle) are more mathematically enticing objects for describing dynamics but practical details of their implementation haven't been worked out yet.
+
+### Phase space as a contangent bundle
+
+Phase space in classical mechanics is typically defined as a contangent bundle.
+Cotangent bundles are, however, hard to use for computation.
+So we select Riesz representers of cotangent vectors according to the metric of our manifold.
+In this way we get an isomorphism between tangent and cotangent bundles.
+
+Note that a generic description of a phase space doesn't require a Riemannian metric.
+Generic tools come from symplectic geometry which imposes a less rich structure.
+
+### Forward Euler? Really? {#sec-forward-euler}
```

Comment:
@kellertuer have you maybe experimented with section cross-links? This doesn't seem to work. I know Quarto docs suggest just using `@sec-foward-euler` and go with the default style but maybe we could have arbitrary text in a link?

Reply:
I would just use Documenter.jl syntax?

Reply:
Something like 
```
### `[Forward Euler? Really?](@ref sec-forward-euler)`{=commonmark}
```
?

Reply:
nearly, the section definitions get `@id`s.